extensions/amp-analytics/0.1/amp-analytics.js

```diff
@@ -162,10 +166,21 @@ export class AmpAnalytics extends AMP.BaseElement {
   layoutCallback() {
     // Now that we are rendered, stop rendering the element to reduce
     // resource consumption.
+    this.element.getResources().setOwner(this.element,
```

Comment:
To clarify, if this is not the correct way to prevent that, please let me know. We discussed it while you were out Friday and others were not familiar with this, but was told that there is not another callback ("destroyCallback" or something) where the current contents of unlayoutCallback would be more appropriately located.